### ISSUE TITLE
Await Reaction [EVENT]

### ISSUE THREAD

**Issue by Naiym223** [dbm-network/mods#729]
```
     const { author } = message
                ^

TypeError: Cannot destructure property 'author' of 'message' as it is undefined.
    at ReactionCollector.msg.awaitReactions.max.max [as filter] (C:\Users\prest\Desktop\BobbyBot\actions\await_reaction_call_action_MOD.js:215:17)
    at ReactionCollector.handleCollect (C:\Users\prest\Desktop\BobbyBot\node_modules\discord.js\src\structures\interfaces\Collector.js:92:46)
    at Client.emit (events.js:326:22)
    at MessageReactionAdd.handle (C:\Users\prest\Desktop\BobbyBot\node_modules\discord.js\src\client\actions\MessageReactionAdd.js:44:17)
    at Object.module.exports [as MESSAGE_REACTION_ADD] (C:\Users\prest\Desktop\BobbyBot\node_modules\discord.js\src\client\websocket\handlers\MESSAGE_REACTION_ADD.js:4:37)
    at WebSocketManager.handlePacket (C:\Users\prest\Desktop\BobbyBot\node_modules\discord.js\src\client\websocket\WebSocketManager.js:386:31)
    at WebSocketShard.onPacket (C:\Users\prest\Desktop\BobbyBot\node_modules\discord.js\src\client\websocket\WebSocketShard.js:436:22)
    at WebSocketShard.onMessage (C:\Users\prest\Desktop\BobbyBot\node_modules\discord.js\src\client\websocket\WebSocketShard.js:293:10)
    at WebSocket.onMessage (C:\Users\prest\Desktop\BobbyBot\node_modules\ws\lib\event-target.js:125:16)
    at WebSocket.emit (events.js:314:20)
```

![image](https://user-images.githubusercontent.com/40970290/124698229-62ec4e80-deb6-11eb-9912-cfa5ec0b8ef2.png)

**Comment by Szabbiii** [comment]
I've got the same problem yesterday, so i just removed line 215 and everything works now :D

**Comment by RediPanda** [comment]
@Szabbiii Please refrain from posting solutions here considering I know that your JavaScript knowledge isn't strong. Just because you can cross the highway once doesn't mean you can do it again.

@Naiym223 Mondon has referenced a PR meaning an official patch has been submitted and it's only waiting for a Mod Maintainer to review and approve.

**Comment by almeidx** [comment]
Fixed in #730
